api/settings.py

```diff
@@ -164,7 +164,8 @@
 PROXY = get_base_config(RAG_FLOW_SERVICE_NAME, {}).get("proxy")
 PROXY_PROTOCOL = get_base_config(RAG_FLOW_SERVICE_NAME, {}).get("protocol")
 
-DATABASE = decrypt_database_config(name="mysql")
+DATABASE_TYPE = os.getenv("DB_TYPE", 'mysql')
```

Comment:
It looks like `mysql` is used here. Should it be `postgres` instead?

Reply:
I'm sorry, I misunderstood. Yes, it should be `postgres`. If `postgres`  will be used as the default storage.